Makefile

```
@@ -5,20 +5,25 @@
 BUILDDIR ?= doc/_build
 DOC_TAG ?= development
 SPHINXOPTS ?= -q
+KCONFIG_TURBO_MODE ?= 0
 
 # Documentation targets
 # ---------------------------------------------------------------------------
 clean:
 	rm -rf ${BUILDDIR}
 
-htmldocs:
-	mkdir -p ${BUILDDIR} && cmake -GNinja -DDOC_TAG=${DOC_TAG} -DSPHINXOPTS=${SPHINXOPTS} -B${BUILDDIR} -Hdoc/ && ninja -C ${BUILDDIR} htmldocs
-
 htmldocs-fast:
-	mkdir -p ${BUILDDIR} && cmake -GNinja -DKCONFIG_TURBO_MODE=1 -DDOC_TAG=${DOC_TAG} -DSPHINXOPTS=${SPHINXOPTS} -B${BUILDDIR} -Hdoc/ && ninja -C ${BUILDDIR} htmldocs
+	${MAKE} htmldocs KCONFIG_TURBO_MODE=1
+
+htmldocs pdfdocs doxygen: configure
+	ninja -C ${BUILDDIR} $@
```

This removes the need to know the build tool, and simply rely on the generator used during configure.

```suggestion
	cmake --build ${BUILDDIR} -- $@
```

---

Applied, thanks!